spec/query_spec.cr

```diff
@@ -104,7 +104,7 @@ describe Avram::Query do
       user = (user_query = UserQuery.new).first?
       user.should_not be_nil
       user.not_nil!.name.should eq "First"
-      user_query.query.statement.should eq "SELECT #{User::COLUMN_SQL} FROM users ORDER BY id ASC LIMIT 1"
+      user_query.query.statement.should eq "SELECT #{User::COLUMN_SQL} FROM users ORDER BY users.id ASC LIMIT 1"
```

Comment:
This is more accurate. Referencing the table means it is less likely to get confused with a different table's id column